airbyte-integrations/connectors/source-snowflake/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,25 @@
+{
+    "streams": [
+        {
+            "stream": {
+                "name": "INTEGRATION_TEST.TEST_TABLE",
+                "json_schema":{},
+                "supported_sync_modes": [
+                    "full_refresh"
+                ],
+                "source_defined_cursor": null,
+                "default_cursor_field": null,
+                "source_defined_primary_key": [
+                    [
+                        "ID"
+                    ]
+                ],
+                "namespace": null
+            },
+            "sync_mode": "full_refresh",
+            "cursor_field": null,
+            "destination_sync_mode": "append",
+            "primary_key": null
```

Comment:
If the `source_defined_primary_key ` is defined, the `primary_key` should be the same.

Reply:
Done